.travis.yml

```diff
@@ -11,15 +11,11 @@ script:
 - bundle exec rubocop --config .rubocop.yml
 deploy:
   provider: heroku
-  api_key:
-    secure: WTzZWZUb5dM2rXYu/Suqs9OhIi2WsET/bX9kGMuLNhV5emMowvHeLtFzQeCXxCbkeb+Hcbsc/pw6PectzlfDXe/42ydOfuLlHTm8G39sR2LzV/YTE+CTJ8RLA6d94MryNBbnpC5QdYpAklkIflR4ZECIcyqJjdaIKGClOF2QGin8BOhRDHiu2nAh8cnGd/8OT15x1ZvYNIQxqtFr/H0l/fNgUPDbuiCvh4rR2HRhXNDUB3CsvZhuUeoyDabbp1HXRIDxE8z389isRH7zOHJ82nY231536isEZ+Da65EqOAloZVDxrttWzR9SkzueTunTQFNNR2g0aQaNRWBJ4AvA3z75PP3nagZXFxiSJRv2XUPmFYWIDEb0gflh86LMUFb6DGWMiIh9GJwBx0wzPXf4uTHOMJxT7YPGddkK64GTHoloL/KnAqtyvgfte4csCXFqKnO3D5MRSrfI4AWpW5bQUzcwFdAVNqOvhKe+bKWJ/EKABfM42sE5IS4Me/jxo26lpkwN/weLVV8CkGlYoytr9Z7LI0OtIUdktiHCg7vwsonbsFuMDXaCXPFaAeGZRVlzGnnmcCnTZdUKyoaHBc/Lokch8GBVLo/zFze7XYhtYcXcmEm3xdn6trKI2s1FBq6aGa1GMSU7Lw7PuXPMeiXdA70QNOVg3GZEf87/A10vJGQ=JFlb9xTKZWsSqYR+BPzcQSDeQZNupDIV9Yrg4FptneAQ84xRoI06n8EEVefSEKenL+MLKawy3LbdhiGGr8sO5KAlJoRUoKabyFbT9/CNu1FxtxHbuu8UXdeUC3jrqbI44+V20o/AF+Lufm5+4dzv1bw3Wjn46qvElCdSZnCrm/RfBqTYtlI6vxcnHgsOl7LTlyQJgOoygqZyNfbADKEk08Rj5wGYFMnwgSSgx/ZWY82VMb9PsJnzBCPsyYX5oU7jj7KrINiKvrM8adzOTeTGb17IzFCRiimI4jzDUoLSDRvB1yGzYgJ/GxAIif/wrMRDelWKUq1X4vsM23J/m7AWMmftWg7lvSs3+q0kq1GDOUnMKQnq9x5jqyyRsMIob4wza1xUouHTYMLDgK86iAaew6Nm4p6Pq8JyU+6extgHcuQpryzmkyh+bli4DJnRHgiVFL0qX9amrgP4gbA=
   app: infinite-loops-chess
+  api_key:
+    secure: YDr7oHqZ0JRErYoqsJid6qQCQqLZKNMofeK7vNHBmE6NcOhBwpTFJNzGNbRP7+DWWPLrczsWhYlTYR8oqTe/GtMXKQae0UANb2+BOQ29pkjho57iRWImn4p5u3TLxeks3OEJbCPQrCA2Mtzpbu5k5eyjT2x2U1JEcD4NyD7Dd6nZ/4afjBS7wgyI0k9p7NJy2noj8ar6SM5Psfq2Ydz05FCPURmlSSWkmEFmtq/mod9KTti9bu/OWqI9vExPibNljQIaxps1jLCMDGb7GVvUUY/Iq/M9NFLasxhbMmPnZZDyMUOL5nmjCyFCupsrX2Z6X18dPncWKxXugwnsz1BPY+xH6zuI+geQVU1sieanuh3okYTu/TP+iaXkHqKN3FzQ5uB6ZDb84rMwxKO3uK4xRSA3VKrMrvQM0JAs6Fx7VVHlN2jkKMfEiR8NwTrmWd8EU7IPENktib3j7qMLjI6zJZQf8atFqV/UrBzIKrBfvN63KIwopBK3cGahCmBPU2InjA8Fsmh6AbzuBX+NlY2hMtRgsXm2LSZn96Imx2sXgQ/mflOrk1d3I4c4mewGrcIJwszgYG026uu16RotgXIXB33n47w1c7X18W++sPt1TplevYgm0d1Pv71GoBnzSvSlCsTL6WeGlOtWeqFFbjKek9ZhX8Y2Xpis5L5ayszNMQQ=
   on:
-    repo: Infinite-Loops-Firehose/chess
+    branch: master
```

Comment:
Dennis, how important is it for us to specify the repo? 

In this link (https://docs.travis-ci.com/user/deployment/heroku/#Conditional-Deploys), they say that including the repo is important if you want to play nice with forks.

As you can see in the snippet I sent to our Slack channel, I got this to work with both the branch and the repo specified, but then it didn't work to restart at the end.